<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewBrushDialog</class>
 <widget class="QDialog" name="AddNewBrushDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Brush</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid rgb(100,100,100);
border-radius: 5px;
margin-top: 2ex;
}

QGroupBox::title {
subcontrol-origin: margin;
left: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="BrushGraphicsView" name="mBrushView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>350</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Brush Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="mObjectNameLE"/>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="mIconLE">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mIconBrowseBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
min-width: 30px;
}</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="mGenIconCB">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Generate Icon</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Icon</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mAddBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrushGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>brushgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mCancelBtn</sender>
   <signal>pressed()</signal>
   <receiver>AddNewBrushDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mGenIconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>AddNewBrushDialog</receiver>
   <slot>onGenerateIconCB(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mIconBrowseBtn</sender>
   <signal>pressed()</signal>
   <receiver>AddNewBrushDialog</receiver>
   <slot>onIconBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAddBtn</sender>
   <signal>pressed()</signal>
   <receiver>AddNewBrushDialog</receiver>
   <slot>onAddBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onIconBrowse()</slot>
  <slot>onGenerateIconCB(bool)</slot>
  <slot>onAddBrush()</slot>
 </slots>
</ui>
